packages/lesswrong/server/clientIdMiddleware.ts

```diff
@@ -19,13 +19,16 @@ const setHasSeen = ({ clientId, userId }: { clientId: string; userId?: string })
 const isApplicableUrl = (url: string) =>
   url !== "/robots.txt" && url.indexOf("/api/") < 0;
 
+// Chrome now has a max expiration time of 400 days for cookies.
+const CLIENT_ID_COOKIE_EXPIRATION_SECONDS = 365 * 24 * 60 * 60;
```

Comment:
Is there any benefit to doing this? The docs say that Chrome doesn't reject the cookie if the date is too far in the future, it just sets the expiry to 400 days itself, and we could still have the longer expiry date in non-Chrome browsers.

Reply:
Sure. Raised it to 10y (as it was before) with a comment that it will likely wind up capped.